templates/injector-network-policy.yaml

```
@@ -10,6 +10,7 @@ apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
   name: {{ template "vault.fullname" . }}-agent-injector
+  namespace: {{ .Release.Namespace }}
```

To ensure the namespace can be overridden like the other changes from #909, I think it makes sense to use the template here:

```suggestion
  namespace: {{ include "vault.namespace" . }}
```